sdk/identity/azure-identity/README.md

```diff
@@ -238,177 +257,166 @@ argument but defaults to the authority matching VS Code's "Azure: Cloud" setting
     </tr>
     <tr>
-      <td><code><a href="https://docs.microsoft.com/java/api/com.azure.identity.usernamepasswordcredential?view=azure-java-stable">UsernamePasswordCredential</a></code></td>
+      <td><code><a href="https://learn.microsoft.com/java/api/com.azure.identity.usernamepasswordcredential?view=azure-java-stable">UsernamePasswordCredential</a></code></td>
       <td>authenticates a user with a username and password without multi-factored auth</td>
-      <td></td>
       <td><a href="https://github.com/Azure/azure-sdk-for-java/wiki/Azure-Identity-Examples#authenticating-a-user-account-with-username-and-password">example</a></td>
-      <td><a href="https://docs.microsoft.com/azure/active-directory/develop/v2-oauth-ropc">Username + password authentication</a></td>
+      <td><a href="https://learn.microsoft.com/azure/active-directory/develop/v2-oauth-ropc">Username + password authentication</a></td>
     </tr>
   </tbody>
 </table>
 
-### Authenticating via Development Tools
+### Authenticating via development tools
 
 <table style="border: 1px; width: 100%;">
   <caption>Authenticating via development tools</caption>
   <thead>
     <tr>
       <th>credential class</th>
       <th>usage</th>
-      <th>configuration</th>
       <th>example</th>
       <th>reference</th>
     </tr>
   </thead>
   <tbody>
     <tr>
-      <td><code><a href="https://docs.microsoft.com/java/api/com.azure.identity.azureclicredential?view=azure-java-stable">AzureCliCredential</a></code></td>
+      <td><code><a href="https://learn.microsoft.com/java/api/com.azure.identity.azureclicredential?view=azure-java-stable">AzureCliCredential</a></code></td>
       <td>Authenticate in a development environment with the enabled user or service principal in Azure CLI</td>
-      <td><a href="https://github.com/Azure/azure-sdk-for-java/wiki/Set-up-Your-Environment-for-Authentication#sign-in-azure-cli-for-azureclicredential">configuration</a></td>
       <td><a href="https://github.com/Azure/azure-sdk-for-java/wiki/Azure-Identity-Examples#authenticating-a-user-account-with-azure-cli">example</a></td>
-      <td><a href="https://docs.microsoft.com/cli/azure/authenticate-azure-cli">Azure CLI authentication</a></td>
+      <td><a href="https://learn.microsoft.com/cli/azure/authenticate-azure-cli">Azure CLI authentication</a></td>
+    </tr>
+    <tr>
+      <td><code><a href="https://learn.microsoft.com/en-us/java/api/com.azure.identity.azurepowershellcredential?view=azure-java-stable">AzurePowerShellCredential </a></code></td>
+      <td>Authenticate in a development environment with the enabled user or service principal in Azure PowerShell</td>
+      <td><a href="https://github.com/Azure/azure-sdk-for-java/wiki/Azure-Identity-Examples#authenticating-a-user-account-with-azure-powershell">example</a></td>
+      <td><a href="https://learn.microsoft.com/en-us/powershell/azure/?view=azps-8.3.0">Azure PowerShell documentation</a></td>
```

Comment:
Omit the querystring, as we always want to display guidance for the latest version
```suggestion
      <td><a href="https://learn.microsoft.com/powershell/azure/">Azure PowerShell documentation</a></td>
```